mmv1/templates/terraform/examples/bigquery_bigquery_table.tf.erb

```diff
@@ -3,6 +3,7 @@ resource "google_bigquery_dataset" "test" {
 }
 
 resource "google_bigquery_table" "test" {
+	deletion_protection = false
```

Comment:
This file is self-consistent spacing-wise, but you could consider changing it to 2-space indentation to be consistent with other files. Not necessary, though!